requirements/pip.txt

```diff
@@ -84,7 +84,7 @@ django-textclassifier==1.0
 django-annoying==0.10.4
 django-messages-extends==0.6.0
 djangorestframework-jsonp==1.0.2
-django-taggit==0.24.0
+django-taggit==1.0.0
```

Comment:
Removed support for python 2 https://github.com/jazzband/django-taggit/blob/master/CHANGELOG.rst#100-2019-03-17. The other changes are from stuff we don't use